### DISCUSSION TITLE
Provide lualine configuration example

### DISCUSSION THREAD

**Discussion by prescientmoon** [m4xshen/hardtime.nvim#69] [Q&A]
Hii!

The readme mentions using lualine to display the hint. Is there any example of how to do that? 

Thanks in advance! ^-^

**Comment by m4xshen** [comment]
Just install lualine.nvim and set `'showmode'` to false and then it should work.

```lua
-- example: using lazy.nvim
{
    "nvim-lualine/lualine.nvim",
    dependencies = { "nvim-tree/nvim-web-devicons" },
    opts = {},
    init = function()
       vim.opt.showmode = false
    end,
 }
```

**Comment by prescientmoon** [comment]
Thanks for the swift response!

What does `showmode` solve here? Judging by `:h showmode` this has no effect when `cmdheight=0` (which is what I am using).

**Comment by m4xshen** [comment]
This method is for user using `cmdheight=1`. If you are using `cmdheight=0` then you should use the third method:

> - Use nvim-notify to display hint messages on the right top corner instead of commandline.
